Beety.Domain/Enums/Enums.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Beety.Domain.Enums
+{
+    public abstract class Enums
+    {
+        public enum ServiceCategory : int
+        {
+            Barbershop = 1
```

Comment:
i think that is bad idea to use enum for category

Reply:
to start normally